packages/tocco-ui/src/Preview/Preview.js

```diff
@@ -0,0 +1,54 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+
+/**
+ * Can be used to show previews of any kind of file. Therefore an Url to the thumbnail and to the file must be provided.
+ */
+const Preview = props => {
+  const onClick = () => {
+    if (typeof props.onClick === 'function') {
+      props.onClick(props.srcLink, props.thumbnailLink)
+    }
+  }
+
+  const figure = (
+    <figure>
+      <img alt={props.alt} className="thumbnail" src={props.thumbnailLink} onClick={onClick}/>
+      {props.caption && <figcaption className="description">{props.caption}</figcaption>}
+    </figure>
+  )
+
+  return (
+    <div className="tocco-preview">
+      {props.downloadOnClick ? <a href={props.srcLink} alt={props.alt} download>{figure}</a> : figure}
+    </div>)
+}
+
+Preview.propTypes = {
+  /**
+   * The url to the image
```

Comment:
*The url to the document (would keep component generic and allways talk about documents rather than image only)